src/main/java/com/lambdaschool/oktafoundation/controllers/OrganizationController.java

```diff
@@ -52,6 +52,22 @@
             return new ResponseEntity<>(og.getApplications(), HttpStatus.OK);
         }
 
+        @PreAuthorize("hasAnyRole('ADMIN')")
+        @GetMapping(value = "/org/name/{orgName}", produces = "application/json")
+        public ResponseEntity<?> getOrgByName(@PathVariable String orgName)
+        {
+            Organization o = orgService.findByName(orgName);
+            return new ResponseEntity<>(o, HttpStatus.OK);
+        }
+
+        @PreAuthorize("hasAnyRole('ADMIN')")
+        @GetMapping(value = "/org/name/like/{orgName", produces = "application/json")
```

Comment:
Still seeing a typo here, missing a closing bracket after orgName

Reply:
I see that typo as well. Also, do we want only admin's to be able to GET information about orgs? I'd think regular users or even anonymous users should be able to see what the org's are?